### ISSUE TITLE
ERROR portable_endian.h:44:72: error: missing binary operator before token "("

### ISSUE THREAD

**Issue by FREEWING-JP** [hoene/libmysofa#215]
hrtf/portable_endian.h:44:72: error: missing binary operator before token "("

https://github.com/hoene/libmysofa/commit/7cfdd0960476822a0a75aeca388da1508a1d400c
```
44 | #elif defined(__DragonFly__) || defined(__FreeBSD__) || defined defined(__NetBSD__) || defined(__OpenBSD__)
```
```
Maybe
defined defined(__NetBSD__)
is
defined(__NetBSD__)
```

**Comment by FREEWING-JP** [comment]
https://github.com/hoene/libmysofa/commit/ff172d7ad4cf48233df893b366eaefe8353760f1

**Comment by hoene** [comment]
fixed by #216
